capplets/time-admin/data/mate-time-admin.desktop.in

```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
+Name=Time And DateManager
```

Comment:
```patch
diff --git a/capplets/time-admin/data/mate-time-admin.desktop.in b/capplets/time-admin/data/mate-time-admin.desktop.in
index 26bff82..3829dc9 100644
--- a/capplets/time-admin/data/mate-time-admin.desktop.in
+++ b/capplets/time-admin/data/mate-time-admin.desktop.in
@@ -1,13 +1,11 @@
 [Desktop Entry]
-Name=Time And DateManager
-Name[zh_CN]=时间和日期管理
-Comment=Manage Time And Date 
-Comment[zh_CN]=管理时间和日期。
+_Name=Time And Date
+_Comment=Manage time and date
 Terminal=false
 Type=Application
 Exec=mate-time-admin
 Icon=mate-times-admin
 StartupNotify=true
-Categories=GTK;Settings;X-MATE-PersonalSettings;
-Keywords=mate-control-center;MATE;personal;information;
+Categories=GTK;Settings;HardwareSettings;
+Keywords=mate-control-center;MATE;clock;date;time;preferences;
 OnlyShowIn=MATE;
```

Reply:
Don't use Title Case for Comment in mate-time-admin.desktop.in:
```shell
$ git grep "Comment="
capplets/about-me/mate-about-me.desktop.in:_Comment=Set your personal information
capplets/accessibility/at-properties/mate-at-properties.desktop.in:_Comment=Choose which accessibility features to enable when you log in
capplets/appearance/data/mate-appearance-properties.desktop.in:_Comment=Customize the look of the desktop
capplets/appearance/data/mate-theme-installer.desktop.in:_Comment=Installs themes packages for various parts of the desktop
capplets/appearance/theme-save.c:               "Comment=%s\n"
capplets/default-applications/mate-default-applications-properties.desktop.in:_Comment=Select your default applications
capplets/display/mate-display-properties.desktop.in:_Comment=Change resolution and position of monitors and projectors
capplets/keybindings/mate-keybinding.desktop.in:_Comment=Assign shortcut keys to commands
capplets/keyboard/mate-keyboard.desktop.in:_Comment=Set your keyboard preferences
capplets/mouse/mate-settings-mouse.desktop.in:_Comment=Set your mouse preferences
capplets/network/mate-network-properties.desktop.in:_Comment=Set your network proxy preferences
capplets/time-admin/data/mate-time-admin.desktop.in:Comment=Manage Time And Date 
capplets/windows/mate-window-properties.desktop.in:_Comment=Set your window properties
font-viewer/mate-font-viewer.desktop.in:_Comment=View fonts on your system
shell/matecc.desktop.in:_Comment=Configure MATE settings
```
Name and Comment should start with underscore for l10n. Test:
```shell
$ ./autogen.sh --prefix=/usr
$ ./makepot
$ grep "Time And Date Manager" mate-control-center.pot
(changes)
$ ./makepot
$ grep "Time And Date Manager" mate-control-center.pot
```

Remove trailing space in Comment.


*.desktop.in files shouldn't include Name[LOCALE]=TRANSLATION, Comment[LOCALE]=TRANSLATION